crypto/keys/secp256k1/internal/secp256k1/secp256_test.go

```
@@ -225,7 +226,10 @@ func BenchmarkSign(b *testing.B) {
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
-		Sign(msg, seckey)
+		_, err := Sign(msg, seckey)
+		if err != nil {
+			fmt.Println(err)
```

I think the benchmark should fail, instead of just printing an error.